demisto_sdk/commands/json_to_outputs/json_to_outputs.py

```diff
@@ -164,6 +164,10 @@ def parse_json(data, command_name, prefix, verbose=False, interactive=False):
 
         raise ValueError('Invalid input JSON')
 
+    # if data is a list of dictionaries [{'a': 'b', 'c': 'd'}, {'e': 'f'}] -> {'a': 'b', 'c': 'd', 'e': 'f'}
```

Comment:
Please also add a comment stating that in case of identical keys, the type will be determined by the last one